src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -35,16 +35,29 @@ export default class SaveInProgressIntro extends React.Component {
           </div>
         );
       } else if (prefillAvailable) {
-        alert = (
-          <div>
-            <div className="usa-alert usa-alert-info schemaform-sip-alert">
-              <div className="usa-alert-body">
-                <strong>Note:</strong> Since you’re signed in to your account, we can prefill part of your application based on your account details. You can also save your form in progress, and come back later to finish filling it out.
+        if (this.props.formId === '21P-527EZ') {
+          alert = (
+            <div>
+              <div className="usa-alert usa-alert-info schemaform-sip-alert">
+                <div className="usa-alert-body">
+                  <strong>Note:</strong>  If you’re signed in to your account, we can prefill part of your application based on your account details. You can also save your form in progress, and come back later to finish filling it out. You have 60 days from the date you start your pension application to submit the form. After 60 days, the form won't be saved and you'll need to start over.
```

Comment:
Are there supposed to be two spaces after "Note:"? Other similar content seems to only have one.

Reply:
Also, why is this "If you're signed in..."? If the user is getting to this point, aren't they definitely signed in since `prefillAvailable` is truthy?

Reply:
Thanks @U-DON, good catch with the spacing, this has been fixed. As for `prefillAvailable`, that refers to the form, rather than the user.

Reply:
Thanks @U-DON, good catches, both fixed.